### DISCUSSION TITLE
Need help to understand

### DISCUSSION THREAD

**Discussion by xalteropsx** [huggingface/diffusers#8559] [General]
@tolgacangoz  bro how can i use this model https://civitai.com/models/16993/badhandv4-animeillustdiffusion in diffusers  just saw it last day dont know how it work

**Comment by tolgacangoz** [comment]
It is said that it is a negative text embedding created by the textual inversion method. [In the documentation, there is a usage example on how to use a negative text embedding.](https://huggingface.co/docs/diffusers/main/en/using-diffusers/loading_adapters#textual-inversion) Could you look over there first, and share your results here; then we can continue the discussion.

**Reply by xalteropsx** [reply]
need to do dinner bro will back soon and read it >.</

**Reply by xalteropsx** [reply]
alright back bro i will check it now sorry was late

**Reply by xalteropsx** [reply]
work nice but look like hira fix will be much better approach but i wonder this prompt is works better too 
@tolgacangoz  thanks alot bro again >.</ u rock 
(((internal-organs-outside-the-body, internal-organs-visible, anatomy-description, unprompted-nsfw ,worst-human-external-anatomy, worst-human-hands-anatomy, worst-human-fingers-anatomy, worst-detailed-eyes, worst-detailed-fingers, worst-human-feet-anatomy, worst-human-toes-anatomy, worst-detailed-feet, worst-detailed-toes, camera, smartphone, worst-facial-details, ugly-detailed-fingers, ugly-detailed-toes,fingers-in-worst-possible-shape, worst-detailed-eyes, undetailed-eyes, undetailed-fingers, undetailed-toes))) // but i do understand they save emedding for short word activate
